python-sdk/src/astro/files/locations/azure/wasb.py

```diff
@@ -89,3 +93,27 @@ def openlineage_dataset_name(self) -> str:
         https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
         """
         return urlparse(self.path).path
+
+    def databricks_auth_settings(self) -> dict:
+        """
+        Required settings to transfer files in/to Databricks.
+
+        Reference:
+        https://docs.databricks.com/external-data/wasb-blob.html
+
+        :return: A dictionary of settings keys to settings values
+        """
+        parsed_url = urlparse(self.path)
+        container_name = parsed_url.netloc
+        account_name = self.hook.get_conn().account_name
```

Comment:
Is this the only way to get an azure key or are there other ways a user might get these? I just want to understand if there are configurations a user might have that would break this.

Reply:
We explored a few paths to authenticate and agreed that we could start with one - and if the community is interested in more, we can extend the approach.